src/components/auth-form/auth-form-button/auth-form-button.styles.js

```diff
@@ -2,8 +2,13 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles((theme) => ({
   authBtn: {
-    border: '1px solid black',
-    borderRadius: '0px',
+    fontFamily: 'Open Sans',
```

Comment:
We have typography.styles file, which have default styles for all text
To not repeat all this styles you should  use variant = 'here you need to chose what suits your styles (h1/body/title)'